sdks/python/src/opik/evaluation/metrics/heuristics/levenshtein_ratio.py

```diff
@@ -39,23 +39,30 @@ def __init__(
         self._case_sensitive = case_sensitive
 
     def score(
-        self, output: str, reference: str, **ignored_kwargs: Any
+        self, output: str, expected_output: Optional[str] = None, reference: Optional[str] = None, **ignored_kwargs: Any
```

Comment:
The renaming here is also debatable, I'm not sure that in practice levenshtein distance is necessary calculated against expected output, I can think of scenarios where someone might want to compare 2 difference LLM outputs and none of them is considered an "expected" one.

Anyway, in the rest of this code review, I consider these renamings as something that we can go with. But I think this question is worth paying attention to.

Reply:
Reverted this changes, will tackle separatly